cmd/mistryd/job.go

```diff
@@ -359,3 +353,55 @@ func (j *Job) BootstrapBuildDir(fs filesystem.FileSystem, log *log.Logger) (bool
 func (j *Job) RemoveBuildDir(fs filesystem.FileSystem, log *log.Logger) error {
 	return fs.Remove(j.ReadyBuildPath)
 }
+
+// ReadLogs returns the job's logs
+func (j *Job) ReadLogs() ([]byte, error) {
+	return ReadJobLogs(j.ReadyBuildPath)
+}
+
+// GetBuildInfo returns the job's build info
+func (j *Job) GetBuildInfo(readLogs bool) (types.BuildInfo, error) {
```

Comment:
I think it's more intuitive to have do a `j.ReadBuildInfo()` that just populates `j.BuildInfo`.